### PR TITLE
Increase default refresh rate in Meter widget

### PR DESCRIPTION
**Pull request by n0toose**
I'm assuming that most workstations should be equipped to handle this.

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

### CODE REVIEW

**Comment by n0toose** [comment]
I did not change this comment because it was still valid and felt like keeping it would be good advice for anyone who may potentially face any sort of issue with the Meter.

https://github.com/tenacityteam/tenacity/blob/f198e8510dc4badf98917c44e8a417e538700f52/src/widgets/Meter.cpp#L2020

**Comment by HeyBanditoz** [comment]
I opened a similar issue upstream, would it be simple to adjust this one too? https://github.com/audacity/audacity/issues/1278

**Comment by n0toose** [comment]
@HeyBanditoz I actually made this change based on your Issue, I'm currently considering how I can implement that on there as well.

**Review by nm17 — DISMISSED** [review]
LGTM, and (in my opinion) 60 should suffice

File: src/widgets/Meter.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
   Dominic Mazzoni
   Vaughan Johnson
 
-  2004.06.25 refresh rate limited to 30mS, by ChackoN
+  2004.06.25 refresh rate limited to 60mS, by ChackoN
```
Comment (nbsp) [inline src/widgets/Meter.cpp]: Nitpick: refresh rate is measured in `Hz`, not `mS`

**Review by nbsp — APPROVED** [review]

**Review by Semisol — APPROVED** [review]

**Pull request merged**